packages/gatsby/src/bootstrap/load-plugins/validate.js

```diff
@@ -214,8 +215,25 @@ const handleMultipleReplaceRenderers = ({ apiToPlugins, flattenedPlugins }) => {
   return flattenedPlugins
 }
 
+function warnOnIncompatiblePeerDependency(name, packageJSON) {
+  // Note: In the future the peer dependency should be enforced for all plugins.
+  const gatsbyPeerDependency = _.get(packageJSON, `peerDependencies.gatsby`)
+  if (
+    gatsbyPeerDependency &&
+    !semver.satisfies(gatsbyVersion, gatsbyPeerDependency)
+  ) {
+    reporter.warn(
```

Comment:
yeah i wasn't satisfied with the fix proposal as well but couldn't find better. i am more inclined to avoid it and stop at second line (and yeah i need to mention gatsby :D )

Reply:
i changed the wording.